File: java/kode program/Tumbuhan.java
// Subclass Tumbuhan, turunan dari MakhlukHidup
public class Tumbuhan extends MakhlukHidup {
    private String jenis;
    private int tinggi;

    public Tumbuhan(int id, String nama, int umur, String jenis, int tinggi) {
        super(id, nama, umur);
        this.jenis = jenis;
        this.tinggi = tinggi;
    }

    @Override
    public String getInfo() {
        return String.format("Tumbuhan| ID: %d | Nama: %s | Umur: %d | Jenis: %s | Tinggi: %d",
                this.id, this.nama, this.umur, this.jenis, this.tinggi);
    }
}

File: python/kode program/tumbuhan.py
from makhluk_hidup import MakhlukHidup

# ============================================
# Kelas Tumbuhan (turunan dari MakhlukHidup)
# ============================================
# Tumbuhan.py


# Kelas turunan dari MakhlukHidup
class Tumbuhan(MakhlukHidup):
    def __init__(self, id, nama, umur, jenis, tinggi):
        super().__init__(id, nama, umur)
        self.jenis = jenis
        self.tinggi = tinggi

    def tampilkan(self):
        # Override untuk menampilkan atribut khusus Tumbuhan
        return f"[Tumbuhan] {super().tampilkan()} {self.jenis:<10} {self.tinggi:<3} m"
